src/main/java/nl/workingtalent/bookrental/controller/BookController.java

```diff
@@ -69,8 +84,38 @@ public Map<String, String> delete(@RequestHeader(name = "Authorization") String
 			throw new ResponseStatusException(HttpStatus.UNAUTHORIZED, "Invalid permissions for deleting book");
 		}
 
-		bookRepo.deleteById(id);
+		Book book = bookRepo.findById(id).get();
```

Comment:
Hier verwijder je op boek niveau, maar ik kan me voorstellen dat je alleen 1 copy wilt verwijderen als die stuk is ofzo

Reply:
Klopt, maar dat was niet de beschrijving van de taak zover ik het begreep. Individuele copies toevoegen en deleten is een mega taak ensich, kan het wel op de backlog toevoegen.